release/train_tests/benchmark/train_benchmark.py

```diff
@@ -230,6 +269,17 @@ def get_metrics(self) -> Dict[str, float]:
                 metrics["validation/local_throughput"] * num_workers
             )
 
+        # Extra time that each worker spends to restore from checkpoint,
+        # which includes downloading the checkpoint, loading the checkpoint,
+        # and skipping through batches that were already processed.
+        restoration_time = (
+            self._metrics["checkpoint/download"].get()
+            + self._metrics["checkpoint/load"].get()
+            + self._metrics["train/iter_skip_batch"].get()
+        )
+        if restoration_time > 0:
+            metrics["checkpoint/restoration_time"] = restoration_time
```

Comment:
Note: We should track "restoration time" in Ray Train by default. This time does not include process group re-init, actor startup, etc. Would be good to sum these two together to get the restoration startup cost.